### DISCUSSION TITLE
aligning 4 tiles as quadrangle

### DISCUSSION THREAD

**Discussion by vkumbhar94** [nikitabobko/AeroSpace#731] [questions-and-answers]
Please redirect me if this is discussed before.
When I open 4 windows of terminal, I would like to align them like quadrangle. I tried but I couldn't succeed, is it possible to align like this or not?

**Comment by Neo3477** [comment]
If you have four windows open next to each other, 1 2 3 and 4....

3 join with right and 2 join with left
